pipe.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env sh
+
+set -e
+
+REPOSITORY_URL=${REPOSITORY_URL:="https://repo.magento.com/"}
+MAGENTO_VERSION=${MAGENTO_VERSION:="magento/project-community-edition:>=2.4.5 <2.4.6"}
+BITBUCKET_CLONE_DIR=${BITBUCKET_CLONE_DIR:="/build"}
+
+GROUP=${GROUP:=""}
+TESTS_PATH=${TESTS_PATH:=""}
+
+composer_setup () {
+  if [ ! -f "composer.lock" ]; then
+      echo "composer.lock does not exist."
+      composer create-project --repository-url="$REPOSITORY_URL" "$MAGENTO_VERSION" /magento2 --no-install
+      cd /magento2
+      composer config repositories.local path $BITBUCKET_CLONE_DIR
```

Comment:
i think this needs to be inside an `if` block otherwise if `COMPOSER_PACKAGES` are not defined, it will fail. this has to be also fixed on the integration test pipe

Reply:
Have added a check and exit if it's not present (should always be present if `composer.lock` is not